src/collective/fingerpointing/logger.py

```diff
@@ -12,33 +14,53 @@
 FORMAT = '%(asctime)s - %(levelname)s - %(message)s'
 
 # by default, the audit log will use the same location used for the event log
-eventlog = getattr(getConfiguration(), 'eventlog', None)
+zopeConf = getConfiguration()
+
+auditlogConf = getattr(
+    zopeConf,
+    'product_config',
+    {}
+).get(
+    'collective.fingerpointing',
+    {}
+)
 
-if eventlog is None:
-    # we are running tests
-    logfile = os.path.join('.', AUDITLOG)
-else:
-    try:
-        logpath = eventlog.handler_factories[0].instance.baseFilename
-        logfolder = os.path.split(logpath)[0]
-        logfile = os.path.join(logfolder, AUDITLOG)
-    except AttributeError:
-        # we are in the debug console
+logfile = auditlogConf.get('audit-log', None)
+if logfile is None:
+    eventlog = getattr(zopeConf, 'eventlog', None)
+    if eventlog is None:
+        # we are running tests
         logfile = os.path.join('.', AUDITLOG)
+    else:
+        try:
+            for factory in eventlog.handler_factories:
+                if type(factory.instance) == FileHandler:
+                    # We found the right Factory.
+                    break
+            logpath = factory.instance.baseFilename
+            logfolder = os.path.split(logpath)[0]
+            logfile = os.path.join(logfolder, AUDITLOG)
+        except AttributeError:
+            # we are in the debug console
+            logfile = os.path.join('.', AUDITLOG)
 
 logger = logging.getLogger(PROJECTNAME)
 logger.setLevel(logging.INFO)
 logger.info('Start logging audit information to ' + AUDITLOG)
 
-# support automatic rotation of audit log files at timed intervals
-# we can later implement a way to make this configurable
-handler = logging.handlers.TimedRotatingFileHandler(
-    logfile, when='midnight', backupCount=30)
+# Use the rotatingfilehandler to rotate at X Bytes,
+# maxBytes can be 0 to never rotate.
+maxBytes = int(auditlogConf.get('audit-log-max-size', 0))
```

Comment:
Users have to use Bytes, not 10M its ok to fail here then (but it will use 10 then).